docs/output.md

```diff
@@ -665,7 +672,7 @@ For further reading and documentation see the [VEP manual](https://www.ensembl.o
 [MultiQC](http://multiqc.info) is a visualization tool that generates a single HTML report summarizing all samples in your project.
 Most of the pipeline QC results are visualised in the report and further statistics are available in the report data directory.
 
```

Comment:
I think it may be helpful to add which QC tools are NOT present in the MultiQC report.